io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "missioncenter",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=org.gnome.Settings",
+    "--system-talk-name=org.freedesktop.systemd1",
+    "--system-talk-name=org.freedesktop.NetworkManager",
+    "--filesystem=xdg-data/flatpak/exports/share:ro",
+    "--filesystem=/var/lib/flatpak/exports/share:ro",
+    "--filesystem=xdg-data/flatpak/app:ro",
+    "--filesystem=/var/lib/flatpak/app:ro",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "missioncenter",
+      "buildsystem": "simple",
+      "build-options": {
+        "append-path": "/run/build/missioncenter/bin",
+        "env": {
+          "RUST_BACKTRACE": "full",
+          "RUSTUP_HOME": "/run/build/missioncenter",
+          "CARGO_HOME": "/run/build/missioncenter/cargo"
+        }
+      },
+      "build-commands": [
+        "rustup default stable-x86_64-unknown-linux-gnu",
+        "cd subprojects/nvtop-* && find ../packagefiles -type f -name 'nvtop-*' -exec sh -c 'patch -p1 < {}' \\;",
```

Comment:
this would be better in a `shell` source nd buildsystem be `meson`

Reply:
Moving this to a shell source doesn't work, since these commands require the sources to the downloaded and extracted/staged